### PR TITLE
docs: SolNetwork.

### PR DESCRIPTION
**Pull request by b4rtaz**
---
name: ['Pull request'
about: A new pull request](docs: SolNetwork.)
---

### Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/SECURITY.md).
- [x] My code is conform the [code style](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/CODE_STYLE.md)
- [x] I have made corresponding changes to the documentation
- [x] I have updated Typescript definitions when needed

### CODE REVIEW

File: packages/solUtils/src/dataTypes/SolNetwork/SolNetwork.ts [hunk 1]
```diff
@@ -26,19 +58,37 @@ export class SolNetwork implements MoralisData {
 
   private constructor(public readonly network: SolNetworkName) {}
 
+  /**
+   * Formats the network to a specific format.
+   * Currently returns a string representing the network.
+   * @example "mainnet"
```
Comment (sogunshola) [inline packages/solUtils/src/dataTypes/SolNetwork/SolNetwork.ts:64]: I think the @example should be how to call the function not what it returns. in this case `network.format()`

File: packages/solUtils/src/dataTypes/SolNetwork/SolNetwork.ts [hunk 2]
```diff
@@ -26,19 +58,37 @@ export class SolNetwork implements MoralisData {
...
+   * @example `network.equals("mainnet")`
+   * @example `network.equals(SolNetwork.create("mainnet"))`
+   */
+  public equals(network: SolNetworkish): boolean {
+    return this.network === SolNetwork.create(network).network;
   }
 
+  /**
+   * @returns a string representing the network.
+   * @example "mainnet"
```
Comment (sogunshola) [inline packages/solUtils/src/dataTypes/SolNetwork/SolNetwork.ts:82]: same as previous comment `network.toJSON()`

File: packages/solUtils/src/dataTypes/SolNetwork/SolNetwork.ts [hunk 3]
```diff
@@ -26,19 +58,37 @@ export class SolNetwork implements MoralisData {
...
+   */
   public toJSON(): string {
     return this.network;
   }
 
+  /**
+   * @returns a string representing the network.
+   * @example "mainnet"
```
Comment (sogunshola) [inline packages/solUtils/src/dataTypes/SolNetwork/SolNetwork.ts:90]: `network.toString()`

**Review by sogunshola — CHANGES_REQUESTED** [review]

**Comment by b4rtaz** [comment]
@sogunshola you're right! Fixed.

**Review by sogunshola — APPROVED** [review]

**Pull request merged**